pulp-glue/pulp_glue/rpm/context.py

```diff
@@ -281,6 +281,8 @@ def preprocess_entity(self, body: EntityDefinition, partial: bool = False) -> En
                     feature=_("customization of the config.repo file"),
                 )
             )
+        if "sqlite_metadata" in body:
+            self.pulp_ctx.needs_plugin(PluginRequirement("rpm", specifier="<=3.24.0"))
```

Comment:
```suggestion
            self.pulp_ctx.needs_plugin(PluginRequirement("rpm", specifier=">=3.24.0", inverted=True, feature=_("...<add somethig useful here>...")))
```

Reply:
The wording of the error message changes from "needed" to "prevents" when using inverted. XD

(In the past, with `min_version` and `max_version` instead of `specifier` this had a subtle influence on the inclusion of the version at the bounds.)

Reply:
Hmm, I was wondering about the invert flag.
Does that error message work?
> Sqlite metadata support was removed from pulp_rpm in version `3.y.z`

Reply:
if you are referring to "feature", then no. (That's why it's not called "error".) Please take a look at how the message is composed.

Reply:
> The wording of the error message changes from "needed" to "prevents" when using inverted. 
I had misunderstood that. I'll use the API description then

Reply:
I urge you to really take a look at how this string snippet is embedded into a sentence. The CLI is all about convenience and UX. A cryptic error message is not helping on that goal.

Reply:
https://github.com/pulp/pulp-cli/blob/main/pulp-glue/pulp_glue/common/context.py#L531